Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#first com
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 09eeacf32416bee17f9e937153ee9355bbd24895)
+++ b/main.py	(date 1777732776590)
@@ -1,1 +1,0 @@
-#first com
\ No newline at end of file
